config/software/openssl-customization.rb

```diff
@@ -61,14 +63,6 @@ def get_sanitized_rbconfig(config)
         f.write("\nrequire 'ssl_env_hack'\n")
         f.write(unpatched_openssl_rb)
       end
-    else
-      embedded_ruby_lib_dir  = get_sanitized_rbconfig('rubylibdir')
```

Comment:
It wasn't doing anything.  It read a file and write it back out.  I think it used to contains the FIPS modifications before you pulled them into the client.